### ISSUE TITLE
Unable to Settle Transactions on L1 via Sequencer and DAC

### ISSUE THREAD

**Issue by puneet166** [0xPolygon/cdk-validium-node#62]
I have successfully built and deployed all CDK-validium-node components using cdk-validium-binary. Individually, each component (rpc, txManager, sequencer, etc.) is functioning correctly. I am able to perform transactions on L2 (CDK) without any issues.

However, the batch of transactions is not settling on L1 via the sequencer and DAC. Upon inspecting the SequenceSener component of CDK, I encountered the following error:

``` error getting signatures and addresses from the data committee: too many members failed to send their signature ```
``` error when trying to get signature from 0xc920737273a002b541bB1d8298Db197E9CbEd1ae: -32000 unauthorized%!(EXTRA string= ```

### Configuration Details:

CDK-validium-node Version: v0.0.3-RC2

### Issue Analysis:

The error suggests an "unauthorized" access issue when the sequencer attempts to request and access the DAC.

### Steps to Reproduce:

* Deploy CDK

* Perform transactions on L2 (CDK).

* Observe the failure to settle transactions on L1 via the sequencer and DAC.

### Expected Behavior:

Transactions should be successfully settled on L1 via the sequencer and DAC.

### Additional Context:

I have meticulously reviewed the configuration details and ensured that all individual components are working seamlessly. The issue seems to be related to unauthorized access during the DAC interaction by the sequencerSender.

### Additionally, I am attaching screenshots of the logs for both the SequencerSender and CDK Data Availability Node for further reference -
![Screenshot 2024-01-30 at 4 08 58 PM](https://github.com/0xPolygon/cdk-validium-node/assets/48614625/f430ed95-8dd8-4219-87b0-46306e6ec946)
![Screenshot 2024-01-30 at 4 32 41 PM](https://github.com/0xPolygon/cdk-validium-node/assets/48614625/def646b7-e310-41fb-bad1-4bdd5d22eac3)

**Comment by bap2pecs** [comment]
have you registered DAC member node on the zkEVM contract? you will need to send a tx to set it up

**Comment by puneet166** [comment]
Hello @bap2pecs ,

I appreciate your prompt response.

Yes, I have successfully initiated the transaction to set it up. Here is the transaction hash along with the details: [Transaction Link](https://sepolia.etherscan.io/tx/0x4da3b322aa363bf985cd0630a0a57dd82b68dde0237910836d5dd14a572b7c73).

Furthermore, I have diligently followed the instructions provided in the container as a committe member.

A screenshot of the docker compose file created for 
![Screenshot 2024-01-30 at 9 45 29 PM](https://github.com/0xPolygon/cdk-validium-node/assets/48614625/9416ce4b-951f-4ceb-9914-ab28a6a656c3)
the committee member is attached herewith for your reference.

DAC_ADDRESS = "trustedAggregator" address.
DEPLOYER_PRIVATE_KEY = " Deployment/admin private key"

**Comment by bap2pecs** [comment]
I believe you have to config your own address in https://github.com/Snapchain/zkValidium-quickstart/blob/main/config/dac as well as updating the key files there

**Comment by puneet166** [comment]
Yes i updated all key file with this command - ``` docker run -d -v .:/key hermeznetwork/zkevm-node /app/zkevm-node encryptKey --pk <privateKey> --pw testonly -o /key ```
and i also config my address  in config.toml file.

Here is the SS -

![Screenshot 2024-01-30 at 10 03 46 PM](https://github.com/0xPolygon/cdk-validium-node/assets/48614625/665371ea-f27a-4421-89d2-c34a4d0c0667)

**Comment by puneet166** [comment]
Entire CDK was working with this image 'hermeznetwork/cdk-validium-node:develop' approximately 30-45 days ago. I had successfully tested it, and it was functioning perfectly at that time.

**Comment by bap2pecs** [comment]
yeah we recently changed to used versioned images in the quickstart to avoid breaking changes on the `develop` and `latest` images

**Comment by bap2pecs** [comment]
I'd suggest to compare your changes with the https://github.com/Snapchain/zkValidium-quickstart/ repo and see where the diffs are
